app/controllers/concerns/scim_rails/response.rb

```diff
@@ -24,6 +24,23 @@ def json_scim_response(object:, status: :ok, counts: nil)
       end
     end
 
+    # TODO: Cut down on redundant code in this file
```

Comment:
🌵 try to have comments right beside whatever it is they're commenting on.
This one is kinda floating off by itself.
If it's about the method, avoid the extra empty line.
If it's about the class, put it at the top above the class definition